src/js/components/orgs/cards/orgActions.tsx

```diff
@@ -138,23 +140,40 @@ const OrgActions = ({
     const needsReview =
       task.has_unmerged_commits && task.pr_is_open && !task.review_valid;
     let isActive = false;
+    let popoverHeading, popoverBody, popoverKey;
+
     switch (type) {
       case ORG_TYPES.DEV:
         isActive = hasReviewRejected || !task.has_unmerged_commits;
+        popoverHeading = i18n.t('Create a Dev Org');
+        popoverKey = 'tourTaskCreateDevOrg';
+        popoverBody =
+          'A Dev Org is a temporary Salesforce org where you can make changes that you would like to contribute to the Project. To create an Org, make sure you are connected to your Salesforce account with a Dev Hub enabled. Use the drop down menu to delete the Org when you no longer need it.';
```

Comment:
It might be cleaner to define `let popover = null;` above, then do something like:

```suggestion
        popover = (
          <TourPopover
            align="right"
            heading={i18n.t('Create a Dev Org')}
            body={(
              <Trans i18nKey="tourTaskCreateDevOrg">
                A Dev Org is a temporary Salesforce org where you can make changes that you would like to contribute to the Project. To create an Org, make sure you are connected to your Salesforce account with a Dev Hub enabled. Use the drop down menu to delete the Org when you no longer need it.
              </Trans>
            )}
          />
        );
```

Then you can define `popover` differently for a QA org, and use it below with just `{popover}` after the button.